### DISCUSSION TITLE
Run flow from UI & show console output

### DISCUSSION THREAD

**Discussion by jakenuts** [FlowiseAI/Flowise#75] [Ideas]
It's a beautiful tool, thanks so much for your effort. I was hoping there might be a way to run a flow from the UI, and/or a way to view the output from the components. Running from npm/npx I wasn't able to see the output and now that I've built the source I'm unable to reach the api. I can figure that stuff out but run/view output would be a nice addition for new users.

**Comment by HenryHengZJ** [comment]
visibility and monitoring is something that we definitely want to implement in the future! we'll get there 💪

**Reply by jakenuts** [reply]
Spent the day playing around with Zapier, Make, n8n, dark, basically anything with a design surface and an LLM integration and while it turned out to be a big disappointment I'm pleased to report you have time.
